astropy/wcs/wcs.py

```diff
@@ -388,8 +396,8 @@ def __init__(self, header=None, fobj=None, key=' ', minerr=0.0,
                         "header must be a string, an astropy.io.fits.Header "
                         "object, or a dict-like object")
 
-            header_string = header_string.strip()
-
```

Comment:
Yes, you are right. However the absence of this line, and the addition of a new line and a comment have the same effect, in the sense that they create keywords which are not in the WCS standard and I expect `WCS.to_header` to return WCS keywords only. Yeah, I agree it returns a valid header but I wish this kind of behaviour change was handled through the `relax` keyword or not introduced at all. It just adds noise.